Formula/go.rb

```diff
@@ -3,24 +3,14 @@ class Go < Formula
   homepage "https://golang.org"
 
   stable do
-    url "https://dl.google.com/go/go1.10.3.src.tar.gz"
-    mirror "https://fossies.org/linux/misc/go1.10.3.src.tar.gz"
```

Comment:
Should be `https://fossies.org/linux/misc/go1.11.src.tar.gz`, but doesn't exist yet